doc/source/releases/22.05_announce.rst

```diff
@@ -1,12 +1,512 @@
 
-:orphan:
-
 ===========================================================
 May 2022 Galaxy Release (v 22.05)
 ===========================================================
 
+.. include:: _header.rst
 
-Schedule
+Highlights
 ===========================================================
- * Planned Freeze Date: 2022-05-02
- * Planned Release Date: 2022-05-23
+
+**The new History is the default History**
+  See the `22.05 user release notes <22.05_announce_user.html>`__ for details.
+
+**Galaxy is now exclusively a FastAPI application**
+  As announced in the release notes for Galaxy Release 22.01 we have removed support to run Galaxy as a WSGI application.
+  If you are still using uWSGI please consult the `Migrating to Gunicorn documentation <https://docs.galaxyproject.org/en/latest/admin/migrating_to_gunicorn.html>`__.
+
+**Deferred dataset resolution**
+  Optional deferred dataset resolution enables users to select that datasets fetched from URLs or remote file sources should not be fetched into Galaxy's objectstore.
+  Instead these datasets are only fetched when jobs using these datasets are run.
+
+**Enhanced Celery tasks and features**
+  Galaxy can optionally delegate the data upload job to Celery, and Galaxy can run the metadata script in Celery. This results in much shorter runtime for small jobs.
+  To enable this set `enable_celery_tasks` to `true` and ensure that at least one celery worker is started.
+  If Celery tasks are enabled it is also possible to change the datatype for many history items in batch.
+
```

Comment:
```suggestion

**Galaxy has a new database migrations system based on Alembic.**
  The new migrations system is built on top of Alembic. The SQLAlchemy Migrate tool been retired. 
  For usage, see `Galaxy's migrations documentation <https://github.com/galaxyproject/galaxy/blob/release_22.05/lib/galaxy/model/migrations/README.md>`__.
  For an architectural overview, see `pull request #13108 <https://github.com/galaxyproject/galaxy/pull/13108>`__.
```

Reply:
That readme is great, but can you link that into the documentation in doc/source/admin, then link that back here ?